Makefile

```diff
@@ -992,6 +992,9 @@ geojson/albers/state-bounds.json: geojson/albers/states.geojson
 geojson/albers/tile-index.json: geojson/albers/us-10m
 	cat $^/*.geojson | ./reproject-geojson --projection mercator --reverse | node_modules/.bin/tile-index -z 7 -f indexed > $@
 
+geojson/albers/centroid-%: geojson/albers/%
```

Comment:
Might be faster to use the simplified files `geojson/albers/us-10m/%` for input.